index.js

```diff
@@ -61,17 +61,20 @@ const sendRequest = (async (method, params) => {
   })
 })
 
-const decodeTransaction = async (transaction_bytecode) => {
-  return await sendRequest('decoderawtransaction', [transaction_bytecode])
-}
-
 const getTransactionData = async (transaction_hashes) => {
```

Comment:
What do you think about renaming this method to `getDogeTransactionData`. For a moment I was worried that we are changing the behavior for all UTXO exporters, it wasn't obvious that this is Doge API specific.

Reply:
sure